src/app/frontend/logs/component.js

```diff
@@ -62,20 +60,29 @@ export class LogsController {
     /** @export {!backendApi.LogDetails} */
     this.podLogs;
 
-    /** @private {!backendApi.LogSelection}*/
-    this.currentSelection;
-
-    /** @private {!ui.router.$state} */
-    this.state_ = $state;
+    /**
+     * Current pod selection
+     * @export {string}
+     */
+    this.pod;
 
-    /** @export {!Array<string>} */
-    this.containers;
+    /**
+     * Current container selection
+     * @export {string}
+     */
+    this.container;
 
-    /** @export {!backendApi.PodContainerList} - initialized from resolve */
-    this.podContainers;
+    /**
+     * Pods and containers available for selection
+     * @export {!backendApi.LogSources}
+     */
+    this.logSources;
```

Comment:
This is not injected, but 'this.podLogs' is set. I don't see any difference. Not sure about the new mechanism...